src/lib/support/TimeUtils.h

```diff
@@ -176,6 +176,17 @@ extern void ChipEpochToCalendarTime(uint32_t chipEpochTime, uint16_t & year, uin
  */
 extern bool UnixEpochToChipEpochTime(uint32_t unixEpochTime, uint32_t & chipEpochTime);
 
+inline bool ChipEpochToUnixEpochTime(uint32_t chipEpochTime, uint32_t & unixEpochTime)
```

Comment:
Why inline?

Reply:
Without inline, there was  duplicate symbol warning for `ChipEpochToUnixEpochTime`. We can move the function definition to the CPP, if that's preferred.

```
ld: warning: duplicate symbol 'chip::ChipEpochToUnixEpochTime(unsigned int, unsigned int&)' in:
    lib/libNetworkTestHelpers.a(libNetworkTestHelpers.NetworkTestHelpers.cpp.o)
    obj/src/protocols/lib/libChipProtocols.a(libChipProtocols.MessageCounterManager.cpp.o)
```

Reply:
Yes, let's move it to the cpp.